lighthouse-core/gather/gatherers/seo/hreflang.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Gatherer = require('../gatherer');
+
+class Hreflang extends Gatherer {
+  /**
+   * @param {{driver: !Object}} options Run options
+   * @return {!Promise<?Array<string>>} Array with hreflang and href values of all link[rel=alternate] nodes found in HEAD, or null
+   */
+  afterPass(options) {
+    const driver = options.driver;
+
+    return driver.querySelectorAll('head link[rel="alternate" i][hreflang]')
```

Comment:
`querySelectorAll` returns `null` instead of `[]`!? we should fix that 😆

Reply:
My bad, it always returns an array. I've updated the code (and jsdoc) accordingly.

Reply:
Oh ok, no worries!